MapboxDirections/MBRouteStep.swift

```diff
@@ -743,6 +749,12 @@ open class RouteStep: NSObject, NSSecureCoding {
      */
     open let instructionsSpokenAlongStep: [SpokenInstruction]?
     
+    /**
+     :nodoc:
+     Contains an array of `VisualInstruction` used for visually conveying information about a given `RouteStep`.
+     */
+    open let visualInstructionsAlongStep: [VisualInstruction]?
```

Comment:
Rename this property `instructionsDisplayedAlongStep`. This name is more consistent with `instructionsSpokenAlongStep` and also emphasizes that the visual instructions are displayed along the containing step but don’t pertain to the containing step.